fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -739,6 +738,10 @@ private synchronized void receiveInteraction(VehicleLaneChange vehicleLaneChange
         }
     }
 
+    private int convertTime(int time) {
```

Comment:
"convertTime" is a rather abstract name for the method. The reader would need to look into the method to see what's happening. Better use "convertSecToMs" or just "secToMs`.